src/crimson/osd/ec_backend.h

```diff
@@ -33,7 +33,6 @@ class ECBackend : public PGBackend
 		      epoch_t min_epoch, epoch_t max_epoch,
 		      std::vector<pg_log_entry_t>&& log_entries) final;
   CollectionRef coll;
-  crimson::os::FuturizedStore::Shard* store;
```

Comment:
in what situation you need PGBackend::store in crimson? finally we should add it back, right?

Reply:
Please note this commit removes the **duplication** of `store` from `ECBackend`, it doesn't touch `PGBackend`. The EC support I'm implementing uses the `store` from `PGBackend`, exactly like `ReplicatedBackend` does.

Reply:
so it is not a sharded store for EC in crimson, right?  can your implementation use seastore also?

Reply:
No. I think it's literally the same instance (and the same rules) as with `ReplicatedBackend`.